src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs_tab.cljs

```diff
@@ -54,7 +57,32 @@
       (let [server-response (:server-response @state)
             {:keys [configs error-message]} server-response]
         (cond
-          (nil? server-response) [comps/Spinner {:text "Loading configurations..."}]
+          (nil? server-response) [comps/Spinner {:text
+                                                 (if (:mc-for-deletion @state)
+                                                   (let [rm_url
+                                                         (paths/rm-method-configuration-path
+                                                           (:workspace-id props)
+                                                           (:mc-for-deletion @state))
+                                                         url-map {:method "DELETE"
+                                                                  :on-done (fn [{:keys [success? xhr]}]
+                                                                             (if success?
+                                                                               ;((:on-import props))
+                                                                               (swap! state assoc
+                                                                                 :server-response
+                                                                                 nil
+                                                                                 :mc-for-deletion
+                                                                                 nil
```

Comment:
Rather than `assoc`ing with `nil`, you can use `dissoc`: `(swap! state dissoc :server-response :mc-for-deletion)`